package.json

```diff
@@ -4,13 +4,17 @@
     "scripts": {
         "help:help": "echo Prints a list of available scripts and a reminder about how to get more help.",
         "help": "echo Available scripts -- execute \\`npm run help:SCRIPTNAME\\` for help on SCRIPTNAME: && (npm run | grep '^\\s\\s\\S' | grep -v 'help\\|only\\|postinstall\\|prepare')",
-        "help:dev": "run-s -s help:dev:*",
+        "help:dev": "run-s -s help:dev:[1-3]",
         "help:dev:1": "echo Compiles frontscope and starts a server running it.",
         "help:dev:2": "echo Note the server will hot-reload the code from any",
         "help:dev:3": "echo source files as they change, to ease development.",
+        "help:dev:workbench": "run-s -s help:dev:workbench:[1-2]",
+        "help:dev:workbench:1": "echo Compiles frontscope and starts a server running it in workbench mode,",
+        "help:dev:workbench:2": "echo \"with the visualizers in the 'visualizers-workbench' directory loaded.\"",
```

Comment:
Since the help texts are printed out in isolation, you may want to either refer to the 'dev' script for more info (e.g. ` otherwise operates like the 'dev' script.` or just reiterate about the hot-reload, since that's the main piece of information that might not be obvious from just `npm run help dev:workbench`.

Reply:
Done.